ENGAGEHF/VitalsManager/Extensions/HKSample+Graphable.swift

```diff
@@ -0,0 +1,73 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2024 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import ExceptionCatcher
+import Foundation
+import SpeziHealthKit
+
+
+/// Conformance to the Graphable protocol allows HKSamples to be displayed no matter what concrete subclass the instance is
+extension HKSample: Graphable {
+    /// Returns the date associated with the measurement
+    /// For now, assumes each sample is associated with a single point in time, not a period
+    public var date: Date { startDate }
+    
+    
+    /// Returns the Double value corresponding to each quantity in the Sample in the given units, if compatable
+    /// For now, only supports HKQuantitySamples and HKCorrelations
+    public func getDoubleValues(for unitString: String) -> [Double] {
```

Comment:
Is there a way to never have to do this conversion from string to HKUnit in the first place?